examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Calling this in us-west-1 would fail our tests, Jordan recommended commenting it out to ensure that the notebook would be testable (in us-west-1).</p>

Reply:
Changed order and added some text to clarify this